streamr-dev/streamr-platform
# [PLATFORM-1107] Code editor zoom

* Fixes positioning of code editor windows to be centered in window on first open.
* Fixes zoomability of editor by updating `react-ace` and adding some config.
* Removes `brace`, replaces with newer `ace-builds`.
* Adds terrible monkeypatch to fix position of annotations, which otherwise still don't work with css scaling.  

#### To test

* Try code editor windows, e.g. Solidity, Java, check working with zoom/pan.
* Try creating syntax errors and applying to see hover annotations positioned appropriately
